doc/conf.py

```diff
@@ -48,7 +50,7 @@
     category=UserWarning,
     message="Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.",
 )
-
+warnings.filterwarnings("ignore", category=PyVistaDeprecationWarning)
```

Comment:
Added to filter warnings related to Pyvista examples that raise deprecation. I don't think it blocks sphinx, but "pollutes"  logs with tons of `warnings` (see https://github.com/pyvista/pyvista/actions/runs/3858649131/jobs/6577403817#step:11:455)

Reply:
This makes me a little nervous... 

IMO the documentation should not use any deprecated examples. 

I know these deprecations are coming from the examples in the deprecated methods themselves...

Perhaps we should remove the examples entirely from the deprecated methods and make sure the new methods they point to have robust examples

Reply:
You convinced me, I will remove examples from deprecated methods and point to the `Cell` ones.

Reply:
Done